components/kserve/kserve.go

```diff
@@ -132,9 +132,13 @@ func (k *Kserve) ReconcileComponent(cli client.Client, owner metav1.Object, dsci
 				return err
 			}
 		}
-	}
-	if err := deploy.DeployManifestsFromPath(cli, owner, DependentPath, dscispec.ApplicationsNamespace, k.GetComponentName(), enabled); err != nil {
-		return err
+		if err := deploy.DeployManifestsFromPath(cli, owner, DependentPath, dscispec.ApplicationsNamespace, k.GetComponentName(), enabled); err != nil {
```

Comment:
This should be outside `if enabled` condition. Reason being DeployManifestsFromPath function takes the enable value to either `deploy` or `delete` component manifests